apps/web/src/server/functions/auth.ts

```diff
@@ -1,9 +1,15 @@
 import * as Sentry from "@sentry/tanstackstart-react";
 import { createServerFn, type ServerFn, type ServerFnCtx } from "@tanstack/react-start";
-import type { User } from "@tasks/core";
+import { bootstrapUseCases, type User } from "@tasks/core";
+import { getKyselyDb } from "@tasks/db";
 
 type AnyServerFnCtx = ServerFnCtx<any, any, any, any>;
 
+const useCases = bootstrapUseCases({
+  kind: "pg",
+  db: getKyselyDb(Sentry),
+});
```

Comment:
this needs to be refoctor in another file. tasks.ts aside does the same. Create a bootstrap.ts file where we centrilize this.